programs/survival/storage_drawers.sc

```diff
@@ -132,8 +125,6 @@ __load_controllers(tag) ->
 	)
 );
 
-__on_start();
-
```

Comment:
Wait, if you remove this call, then that function is never called mate

Reply:
And that code never runs afaik

Reply:
That is now an event in Carpet (as specified in the pr body).

Reply:
oh Im dumb I didnt read properly lol